hack/e2e/values.venafi-kubernetes-agent.yaml

```diff
@@ -10,4 +10,4 @@ authentication:
 
 extraArgs:
 - --logging-format=json
-- --log-level=2
+- --log-level=6
```

Comment:
Here's  how it looks in Google Logs Explorer

<img width="960" alt="image" src="https://github.com/user-attachments/assets/b4961ad5-4060-47a3-910a-be56e0886343">